src/components/learner-credit-management/AssignmentRowActionTableCell.jsx

```diff
@@ -1,45 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {
-  Icon,
-  IconButton,
-  OverlayTrigger,
-  Stack,
-  Tooltip,
+  Icon, IconButton, OverlayTrigger, Stack, Tooltip,
 } from '@edx/paragon';
 import { Mail, DoNotDisturbOn } from '@edx/paragon/icons';
 
 const AssignmentRowActionTableCell = ({ row }) => {
-  const cancelButtonMarginLeft = row.original.state === 'allocated' ? 'ml-2.5' : 'ml-auto';
+  const learnerStateBool = row.original.learnerState === 'waiting';
+  const cancelButtonMarginLeft = learnerStateBool ? 'ml-2.5' : 'ml-auto';
   return (
-    <div className="d-flex">
-      {row.original.state === 'allocated' && (
-        <>
-          <OverlayTrigger
-            key="Remind"
-            placement="top"
-            overlay={<Tooltip variant="light" id="tooltip-remind">Remind learner</Tooltip>}
-          >
-            <IconButton
-              className="ml-auto mr-0"
-              src={Mail}
-              iconAs={Icon}
-              alt="Remind learner"
+    <Stack direction="horizontal" gap="1">
```

Comment:
I believe this should have `className="justify-content-end"` to force the contents of this table cell to be right-aligned in the table, based on the Figma mocks.

<img width="730" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/059bfcab-a66e-48cc-8bda-61f7401d8d36">

Note that the subscriptions table (for comparison) appears to not be properly right aligned as expected.